docs/basic-operations.md

```diff
@@ -0,0 +1,105 @@
+<!--[metadata]>
++++
+title = "Basic Operations"
+description = "Basic operations using Notary"
+keywords = ["docker, notary, trust, image, signing, repository, cli"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Basic operations using Notary
+
+The notary CLI provides a convenient way of managing trusted collections and the keys associated with them. On this page we describe a high level overview of the functionality of each subcommand. For a quick walk-through of the basic Notary functionality check out [getting started with Notary](gettingstarted.md).
+
+## Creating a new collection of data
+
+Before you create a new collection of data, you have to name it. In notary, each collection of data is identified by a Global Unique Name (GUN). GUNs can be arbitrary, but they have to be unique. An easy way of having unique GUNs is by including any other pre-existing global namespace already available such as DNS. A good example of a GUN is: example.com/user/collection.
+
+After chosing a name, you can initialize a repository by running the `init` command:
+
+```
+$ notary init example.com/user/collection
+```
+
+On initialization, notary creates all the necessary keys needed to sign content for this collection, create the necessary TUF metadata files and push them to the remote notary server. If you already have a root key, notary uses it instead of generating a new one.
```

Comment:
Grammar nitpick:  the tenses of the verbs should match - i.e. "..., creates the necessary TUF..., and pushes them...".